.scrutinizer.yml

```diff
@@ -13,7 +13,7 @@ build:
     nodes:
         analysis:
             environment:
-                php: 7.4.12
+                php: 8.0.18
```

Comment:
Can't we use `8.0.20` https://www.php.net/releases/index.php? Or it's not supported in CI yet? It's probably a better idea to not specify patch version (just `8.0`) Is it possible?

Reply:
Didn't notice that PR was created at May 24.